src/main/scala/Keyboard.scala

```diff
@@ -0,0 +1,37 @@
+object Keyboard extends App {
+  import scala.io.Source
+  import scala.io.StdIn._
+  println("Enter number of character strings to read")
+  val wordAmount = readInt
+  var stringsArray = new Array[String](wordAmount)
+  println("Type that many strings")
+  for (i <- 0 until wordAmount)stringsArray(i) = readLine
```

Comment:
you could do for (i <- 0 until readInt) and then it would read users input for how many times it should read keyboards, saves a line of code and place in memory